testing/jormungandr-testing-utils/src/testing/node/explorer/load/mod.rs

```diff
@@ -153,7 +157,13 @@ impl RequestGenerator for ExplorerRequestGen {
         })
     }
 
-    fn split(self) -> (Self, Option<Self>) {
+    fn split(mut self) -> (Self, Option<Self>) {
+        // Since no transaction will ever be accepted we could split as many times as we want
```

Comment:
This comment is not correct in this context. In this case the explorer queries do not modify the node state and can be run concurrently without problems